src/Sylius/Bundle/UserBundle/Resources/config/serializer/Model.User.yml

```diff
@@ -20,7 +20,7 @@ Sylius\Component\User\Model\User:
     relations:
         - rel: user
```

Comment:
so should I leave the route as it was before? but the it has to be created in some routing.yml file as it doesn't exist.